test/enigma_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require_relative '../lib/enigma'
+
+class EnigmaTest < Minitest::Test 
+
+  def test_it_exists 
+    enigma = Enigma.new
+    assert_instance_of Enigma, enigma
+  end
+  
+  def test_it_can_encrypt_a_message
+    enigma = Enigma.new
+    expected = { encryption: 'keder ohulw', key: '02715', date: '040895' }
+    assert_equal expected, enigma.encrypt('hello world', '02715', '040895')
+
+    expected = { encryption: 'nib udmcxpu', key: '02715', date: '110120' }
+    assert_equal expected, enigma.encrypt('hello world', '02715')
+
+    # expected = { encryption: 'nib udmcxpu', key: '02715', date: '110120' }
```

Comment:
**needs mocks and stubs to test the case where the `encrypt`/`decrypt` methods are called without arguments.**